packages/fxa-admin-server/src/graphql.ts

```diff
@@ -150,6 +150,7 @@ export interface IQuery {
 export interface IMutation {
     unverifyEmail(email: string): boolean | Promise<boolean>;
     disableAccount(uid: string): boolean | Promise<boolean>;
+    recordAdminSecurityEvent(name: string, uid: string): boolean | Promise<boolean>;
```

Comment:
Q: Are these in the right order? Above it looks like `uid, name`&hellip;
```js
public async recordAdminSecurityEvent(
  @Args('uid') uid: string,
  @Args('name') name: string
)
```
and&hellip;
```js
export const RECORD_ADMIN_SECURITY_EVENT = gql`
  mutation recordAdminSecurityEvent($uid: String!, $name: String!) {
    recordAdminSecurityEvent(uid: $uid, name: $name)
  }`;
```


But the order here looks like `name, uid`&hellip;
```js
recordAdminSecurityEvent(name: string, uid: string): boolean | Promise<boolean>;
```

Reply:
Shoot you're right -- thank you for catching that. I'm guessing it didn't cause an error bc they were the same type, but confusing

Reply:
....Nope, graphql seems to be rewriting it. Whenever I change it in `graphql.ts`, it is automatically changed back. I guess this may be one argument for alphabetizing my inputs

Reply:
😂 I reversed the args in all the instances and also the `graphql.ts` file....and now `graphql.ts` is saying it should be the opposite. I have no idea what is happening here

Reply:
I  guess I would say, observing the above, I'm not sure why the fields are auto-reversing so I don't know if they're in the wrong order there or not.

Reply:
Definitely weird behavior. Admiditly I generally don't check the generated code  files. I think it's a non issue because the actual mutation resolves the args by name, e.g. @Args(**'uid'**).   Strange none the less.